### PR TITLE
helm add auditLogConfig option

### PR DESCRIPTION
**Pull request by kmlebedev**

### CODE REVIEW

File: k8s/helm_charts2/values.yaml [hunk 1]
```diff
@@ -273,9 +273,6 @@ filer:
   # Limit sub dir listing size (default 100000)
   dirListLimit: 100000
 
-  # Turn off directory listing
-  disableDirListing: false
```
Comment (chrislusf) [inline k8s/helm_charts2/values.yaml:277]: This is an unrelated change.

Comment (kmlebedev) [reply k8s/helm_charts2/values.yaml:277]: @chrislusf  This is a duplicate of IDE highlighted
https://github.com/chrislusf/seaweedfs/blob/master/k8s/helm_charts2/values.yaml#L263
